releasing.md

```diff
@@ -0,0 +1,31 @@
+# Releasing UgandaEMR Versions 
+
+UgandaEMR leverages the OpenMRS infrastructure on BinTray offered by JFrog, which is a maven based distribution service. This allows UgandaEMR modules to be used by the greater OpenMRS community, if needed and automation of the processes for release management.
+
+The release process uses the maven release plugin 
+
+## Release Process
+
+The process for releasing is as follows:
+
+1. Setup UgandaEMR CI credentials in the local .m2 repository settings 
+2. Create a new clone of the project directly from the module repository, recommended is to name the clone release-openers-module-XXX. This differentiates it from the regular working repositories 
```

Comment:
Which project here am i supposed to clone exactly

Reply:
@solemabrothers that why we are to do it together ;-)

Reply:
Noted